AcadeMeData/test_professor.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from AcadeMeData.models import Professor, University
+
+
+@pytest.fixture
+def generate_professor(generate_university, professor_id=2, name="DR Arnold Schwarzenegger",
+                       description="A cool guy who looked familiar", rate=4.5):
+    university = generate_university
+    professor = Professor.create_professor(professor_id=professor_id,
+                                           name=name,
+                                           university=university,
+                                           description=description,
+                                           rate=rate)
+    return professor
+
+
+@pytest.fixture
+def generate_university(university_id=3, name='The Hebrew University', location="Jerusalem",

```

Comment:
Since `generate_university` is used in both the professor and university tests, I'd add it to a [conftest](https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session) file

Reply:
will do

Reply:
> Since `generate_university` is used in both the professor and university tests, I'd add it to a [conftest](https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session) file

If you could take a look - I tried adding content.py and it didn't work, looked online but couldn't fix the error.